### PR TITLE
{data} [intel/2016a] PostgresSQL 9.5.2 + Postgres-XL 9.5(REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2006.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c3141cde7c08251359cc490a72de5dba for a full test report.

**Comment by wpoely86** [comment]
@boegel please review

File: easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.2-intel-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+osdependencies = [('openssl-devel', 'libssl-dev')]
+
+java = 'Java'
+javaver = '1.8.0_77'
+
+dependencies = [
+    (java, javaver, '', True),
+    ('libreadline', '6.3'),
+    ('zlib', '1.2.8'),
+    ('Python', '2.7.11'),
```
Comment (boegel) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.2-intel-2016a.eb]: include a `%(pyver)s` versionsuffix?

Comment (wpoely86) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.2-intel-2016a.eb]: seems like overkill? We don't do it for java neither.

Comment (boegel) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.2-intel-2016a.eb]: for Python, we do it consistently, I feel it helps in making it clear what other (Python) tools/libraries can be combined with this module

**Comment by boegel** [comment]
I guess this replaces #2713 by @ehiggs?

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2723.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7c051354f227f39a08672753503bc02a for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node2723.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/dbc24b14c49f8d69d7dc3ef65e300908 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node2723.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b046b513510437898452d72306bc08bb for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2006.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/20321a3d2db7c9b42fd83ec795fe3c02 for a full test report.

**Comment by boegel** [comment]
@wpoely86 problems on CentOS7?

File: easybuild/easyconfigs/m/MADlib/MADlib-1.9-intel-2016a-Python-2.7.11.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('PostgreSQL', '9.5.2', versionsuffix),
+]
+
+separate_build_dir = True
+
+sanity_check_paths = {
+    'files': ['bin/madpack'],
```
Comment (boegel) [inline easybuild/easyconfigs/m/MADlib/MADlib-1.9-intel-2016a-Python-2.7.11.eb]: that's it? not libraries?

File: easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.2-intel-2016a-Python-2.7.11.eb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+builddependencies = [
+    ('Bison', '3.0.4'),
+    ('flex', '2.6.0'),
+    ('Perl', '5.22.1'),
+]
+
+configopts = '--with-python --with-openssl'
+
+sanity_check_paths = {
+    'files': ['bin/psql'],
```
Comment (boegel) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.5.2-intel-2016a-Python-2.7.11.eb]: only `psql`, no libraries/headers?

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2685.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/41ebfa8b8d32e9469122902f309d4840 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2685.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c99287fe2ea351ac6594f3be741dc72b for a full test report.

**Comment by wpoely86** [comment]
@boegel please review/merge

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5d3207247e0eed638cb8c42f1e882e45 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**